fabric-api-lookup-api-v1/src/main/java/net/fabricmc/fabric/api/lookup/v1/block/BlockApiLookup.java

```diff
@@ -227,6 +227,13 @@ default A find(World world, BlockPos pos, C context) {
 	 */
 	Class<C> contextClass();
 
+	/**
+	 * Return the provider for the passed block (registered with one of the {@code register} functions), or null if none was registered (yet).
```

Comment:
```suggestion
	 * {@return the provider for the passed block (registered with one of the {@code register} functions), or null if none was registered (yet)}
```
An inline `@return` also puts it as the method return value description.

Reply:
Not sure tbh, this is slightly harder to read in java form, and makes the javadoc redundant. Also, intellij doesn't seem to properly show it (it shows it like a regular `@return`). Probably a good reason not to do this.

Reply:
Eh, shouldn't really use intellj bug as an excuse. This both leaves this statement at the beginning of the javadoc and adds a corresponding "return" section in generated javadoc

Reply:
I find the original javadoc less readable then. Duplicating the description just feels stupid to me, automatic or not.